docs/changelog/changelog_dev.rst

```diff
@@ -25,4 +25,14 @@ Development Changelog
         :tags: general
         :pullreq: 326
 
-        Adds some integration tests with the lastest ResStock.
\ No newline at end of file
+        Adds some integration tests with the lastest ResStock.
+
+    .. change::
+        :tags: validaton
```

Comment:
validation

Reply:
Oops. Ha. Thanks for catching all my hasty typos.